src/nl/Stratego/Bord.java

```diff
@@ -97,9 +107,52 @@ public String toString(){
                     bordstring.append(spelstukString);
                 }
                 bordstring.append("|\n");//Aan het einde komt nog een rechtstreepje en dan een niewline character
-                bordstring.append("+---+---+---+---+---+---+---+---+---+---+\n");
+                bordstring.append("  +---+---+---+---+---+---+---+---+---+---+\n");
             }
-        return bordstring.toString();
+
+        System.out.println(bordstring);
+    }
+
+    //met onderstaande methode wordt het bord geprint teamspecifiek, je geeft dan het team mee in de methode
+
+    public void bordPrinten(int huidigeTeam){
+
+        StringBuilder bordstring = new StringBuilder();
+        bordstring.append("X:  1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10| \n"); //deze coordinaten worden geprint boven het bord
+        bordstring.append("  -----------------------------------------\n"); // dit is een afscheiding van coordinaten tov gevulde matrix
+        bordstring.append("Y:+---+---+---+---+---+---+---+---+---+---+\n");
+        for (int y = 0; y < 10; y++) {
+            int yCoordinaat = y+1; //deze y-coordinaat wordt gedefinieerd zodat deze geprint kan worden als coordinatenstelsel
+            if(yCoordinaat<10) {
+                bordstring.append(" ");//getallen kleiner dan 10, krijgen extra spatie (voor uitlijning)
+            }
+            bordstring.append(yCoordinaat);
+            for (int x = 0; x < 10; x++) { //deze forloop voegt voor ieder vakje de value van het spelstuk toe of een "o" als het vakje leeg is.
+                String spelstukString;
+                if (speelBord[y][x] instanceof Speelstuk) {
+                    Speelstuk speelstuk = (Speelstuk) speelBord[y][x];
+                    int team = speelstuk.getTeam();
+                    int value = speelstuk.getValue();
```

Comment:
De "team" variabele kan je ook direct in de if statement zetten: if (speelstuk.getTeam() == huidigeTeam) {}.
int value kan je dan in die if statement zetten. Dan wordt deze alleen geëvalueerd als het van het huidig team is. Scheelt processor kracht.

Reply:
Dat geldt dan ook voor getValue, ipv value. Ik verander dat dan allebei, scheelt ook weer twee regels